<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>miilestone1</class>
 <widget class="QWidget" name="miilestone1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>miilestone1</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(60, 64, 64);</string>
   </property>
   <property name="title">
    <string>State/City</string>
   </property>
   <widget class="QComboBox" name="stateList">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>45</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(75, 164, 255);
</string>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QLabel" name="cityList1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 155, 255);</string>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QListWidget" name="cityList">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="zipCodeList">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Zipcodes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 155, 255);</string>
    </property>
    <property name="text">
     <string>Zipcodes</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>441</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 64, 64);</string>
   </property>
   <property name="title">
    <string>Business Categories</string>
   </property>
   <widget class="QListWidget" name="businessCategories">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="ZipcodeStats">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 155, 255);</string>
    </property>
    <property name="text">
     <string>Zipcode statistics</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ZipcodeBusinessTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="ZipcodeStats_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 155, 255);</string>
    </property>
    <property name="text">
     <string>Businesses Filtered by category</string>
    </property>
   </widget>
   <widget class="QListWidget" name="businessCategories2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ZipcodeBusinessTable_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="ZipcodeStats_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 155, 255);</string>
    </property>
    <property name="text">
     <string>Businesses Categories</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>871</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(189, 188, 191);</string>
   </property>
   <property name="title">
    <string>Search Business by Zipcode</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Business Zipcode</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zname">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Businesses in Zipcode</string>
    </property>
   </widget>
   <widget class="QListWidget" name="businesses">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Zipcodes_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zipcodes</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="zBusinessTable">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
